RUN3/AliAnalysisTaskAO2Dconverter.h

```diff
@@ -130,7 +131,16 @@ class AliAnalysisTaskAO2Dconverter : public AliAnalysisTaskSE
       kTPCPileUp,
       kTimeRangeCut,
       kEMCALEDCut,
-      kAliEventCutsAccepted
+      kAliEventCutsAccepted,
+      kIsPileupFromSPD,
+      kIsV0PFPileup,
+      kIsTPCHVdip,
+      kIsTPCLaserWarmUp,
+      kTRDHCO,
```

Comment:
Looks very good. Could you add comments behind the TRD ones, so they are clear to everyone?